opendevin/runtime/docker/ssh_box.py

```diff
@@ -690,13 +690,13 @@ def restart_docker_container(self):
             if self.use_host_network:
                 network_kwargs['network_mode'] = 'host'
             else:
-                # FIXME: This is a temporary workaround for Mac OS
+                # FIXME: This is a temporary workaround for Windows where host network mode has bugs.
+                # FIXME: Docker Desktop for Mac OS has experimental support for host network mode
                 network_kwargs['ports'] = {f'{self._ssh_port}/tcp': self._ssh_port}
                 logger.warning(
                     (
-                        'Using port forwarding for Mac OS. '
-                        'Server started by OpenDevin will not be accessible from the host machine at the moment. '
-                        'See https://github.com/OpenDevin/OpenDevin/issues/897 for more information.'
+                        'Using port forwarding till the enable host network mode is out of experimental mode.'
```

Comment:
maybe just add the word "Docker" to it, like "...the 'enable host network mode' of Docker is out..." ?